sdk/python/feast/client.py

```diff
@@ -605,370 +578,21 @@ def get_feature_table(
-                Each feature reference should have the following format:
-                "feature_set:feature" where "feature_set" & "feature" refer to
-                the feature and feature set names respectively.
-                Only the feature name is required.
-            entity_rows: A list of dictionaries where each key is an entity and each value is
-                feast.types.Value or Python native form.
-            project: Optionally specify the the project override. If specified, uses given project for retrieval.
-                Overrides the projects specified in Feature References if also are specified.
-            omit_entities: If true will omit entity values in the returned feature data.
-        Returns:
-            GetOnlineFeaturesResponse containing the feature data in records.
-            Each EntityRow provided will yield one record, which contains
-            data fields with data value and field status metadata (if included).
-
-        Examples:
-            >>> from feast import Client
-            >>>
-            >>> feast_client = Client(core_url="localhost:6565", serving_url="localhost:6566")
-            >>> feature_refs = ["daily_transactions"]
-            >>> entity_rows = [{"customer_id": 0},{"customer_id": 1}]
-            >>>
-            >>> online_response = feast_client.get_online_features(
-            >>>     feature_refs, entity_rows, project="my_project")
-            >>> online_response_dict = online_response.to_dict()
-            >>> print(online_response_dict)
-            {'daily_transactions': [1.1,1.2], 'customer_id': [0,1]}
-        """
-
-        try:
-            response = self._serving_service.GetOnlineFeatures(
-                GetOnlineFeaturesRequest(
-                    omit_entities_in_response=omit_entities,
-                    features=_build_feature_references(feature_ref_strs=feature_refs),
-                    entity_rows=_infer_online_entity_rows(entity_rows),
-                    project=project if project is not None else self.project,
-                ),
-                metadata=self._get_grpc_metadata(),
-            )
-        except grpc.RpcError as e:
-            raise grpc.RpcError(e.details())
-
-        response = OnlineResponse(response)
-        return response
-
     def ingest(
         self,
-        feature_set: Union[str, FeatureSet],
+        feature_table: Union[str, FeatureTable],
         source: Union[pd.DataFrame, str],
+        project: str = None,
         chunk_size: int = 10000,
         max_workers: int = max(CPU_COUNT - 1, 1),
-        disable_progress_bar: bool = False,
-        timeout: int = KAFKA_CHUNK_PRODUCTION_TIMEOUT,
-    ) -> str:
+        timeout: int = BATCH_INGESTION_PRODUCTION_TIMEOUT,
+    ) -> None:
         """
-        Loads feature data into Feast for a specific feature set.
+        Batch load feature data into batch source of a specific feature table.
```

Comment:
We should just say `into a Feature Table`. In the future it would go into the offline store.

Reply:
Updated.